release-kubeapps-package.sh

```diff
@@ -0,0 +1,104 @@
+#! /usr/bin/env bash
+
+# Copyright 2022 the Kubeapps contributors.
+# SPDX-License-Identifier: Apache-2.0
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Variables
+readonly script_name=$(basename "${0}")
+readonly script_dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+readonly template_dir="$script_dir/packaging-templates"
+readonly git_repo="vmware-tanzu/package-for-kubeapps"
+
+# Variables set by get_options.
+version=""
+packaging_version_suffix=""
+
+main() {
+  get_options "${@}"
+  
+  # Commit, tag and create a release for production only.
+  echo "Releasing Kubeapps Carvel package v$version to repo '$git_repo'"
+  create_release "$version" "$packaging_version_suffix"
+}
+
+# Commit, tag and create a release in GitHub.
+create_release() {
+  local version=$1
+  local packaging_version_suffix=$2
+  local version_with_suffix="$version$packaging_version_suffix"
+  local tag="v$version_with_suffix"
+  git add "./$version_with_suffix"
+  git commit -m "Adding $tag files"
+  echo "Committing files and tagging $tag for release"
+  git tag "$tag" -m "$tag"
+  # Pick first remote to push
+  local git_origin="$(git remote | head -n 1)"
+  git push "$git_origin" "tags/$tag"
+
+  echo "Creating release for $tag"
+  gh release create "$tag" "./$version_with_suffix/package.yaml" "./metadata.yaml" "./README.md" \
```

Comment:
Should we be installing the GitHub CLI as well in the container?

Reply:
I've added a comment about it in this PR's description. For the Git+GitHub operations we need valid credentials, which makes things more complex. I decided to have it mostly automated, except this part, that can be improved in a later PR.

Reply:
Ah, ok... but can't we just run the container and pass through it some env vars with the credentials `docker run ... -e XXXX=YYY` ? Anyway, ok with the semi-automated way for now.

Reply:
> can't we just run the container and pass through it some env vars with the credentials

Exactly, that will be done in a later PR. I've added a comment in the [issue](https://github.com/vmware-tanzu/package-for-kubeapps/issues/11#issuecomment-1225575704) and will leave it opened.